diff --git a/cli/tools/upgrade.rs b/cli/tools/upgrade.rs
index d72fb1b0a..d2f1652f5 100644
--- a/cli/tools/upgrade.rs
+++ b/cli/tools/upgrade.rs
@@ -18,9 +18,9 @@ use std::path::PathBuf;
 use std::process::Command;
 
 static ARCHIVE_NAME: Lazy<String> =
-  Lazy::new(|| format!("deno-{}.zip", env!("TARGET")));
+  Lazy::new(|| format!("deno-linux-arm64.zip"));
 
-const RELEASE_URL: &str = "https://github.com/denoland/deno/releases";
+const RELEASE_URL: &str = "https://github.com/deno911/arm64/releases";
 
 pub async fn upgrade(upgrade_flags: UpgradeFlags) -> Result<(), AnyError> {
   let old_exe_path = std::env::current_exe()?;
